src/en/getting-started.md

```diff
@@ -217,33 +229,29 @@ juju status
 When the applications have been installed, the output to the above command will
 look something like this:
 
-![juju status](./media/juju-mediawiki-status.png)
-
-There is quite a lot of information there but the important parts for now are 
-the [Applications] section, which show that Mediawiki and MySQL are installed, 
-and
-the [Units] section, which crucially shows the IP addresses allocated to them.
+![juju status](./media/juju-status-wiki-simple.png)
 
-By default, Juju is secure - you won't be able to connect to any applications 
-unless they are specifically exposed. This adjusts the relevant firewall 
-controls (on any cloud, not just LXD) to allow external access. To make
-our Mediawiki visible, we run the command:
+There is lots of juicy information there! The important parts for now are
```

Comment:
Juicy. I threw that in to see if anyone noticed. Bravo. I'll reword.

Collective nouns such as *information* can be used as singular or plural in British English, there is no preference.